draft-ietf-aipref-vocab.md

```diff
@@ -203,11 +203,144 @@ If arrangements, such as contracts exist that explicitly permit the use of that
 
 The vocabulary does not preclude the use of other specific categories. Any statement of preference based on this vocabulary shall not be interpreted as restricting the use of the work(s) strictly for the purpose of search and discovery as long as no restriction is declared through search-specific means such as {{!RFC9309}}.
 
-## Vocabulary Extensions
+## Vocabulary Extensions {#vocab-extension}
 
 Systems referencing the vocabulary must not introduce additional categories that include existing categories defined in the vocabulary or otherwise include additional hierarchical relationships.
 
 
+# Serialization Format {#format}
+
+This section defines an exemplary serialization format for preferences.
+The format describes how the abstract model could be turned into a Unicode string or sequence of bytes.
```

Comment:
By Unicode, you really mean UTF-8, yes?  If so, then say that since Unicode can be encoded in many ways.

Reply:
Hm, Unicode string has a definition that doesn't match what I wanted: https://www.unicode.org/versions/Unicode16.0.0/core-spec/chapter-3/#G32765

What I'm looking for is probably Unicode text: a term without apparent definition, though Unicode uses it often.  Or maybe "coded character sequence": https://www.unicode.org/versions/Unicode16.0.0/core-spec/chapter-3/#G45138  (which is spectacularly opaque, so I might just use "Unicode text").

WDYT?

Reply:
```suggestion
The format describes how the abstract model could be turned into Unicode text or sequence of bytes.
```